MAD.Procore.StudioHoursUpload/Startup.cs

```diff
@@ -36,6 +37,16 @@ public void ConfigureServices(IServiceCollection serviceDescriptors)
             serviceDescriptors.AddTransient<StudioHourClient>();
         }
 
+        public static void PostConfigure(StudioHourDbContext dbContext, ProcoreConfig procoreConfig, IRecurringJobManager recurringJobManager)
+        {
+            dbContext.Database.Migrate();
+
+            var queueName = procoreConfig.Name.ToLower().Replace(" ", "_");
+
+            recurringJobManager.AddOrUpdate<StudioHourUploadLogProducer>($"{procoreConfig.Name}.{nameof(StudioHourUploadLogProducer)}.ProduceStudioHoursUploadLogs", y => y.ProduceStudioHoursUploadLogs(), CronEveryXHours, null, queueName);
```

Comment:
Use named parameters and remove the CronEveryXHours. Just want to use the default CRON schedule which it defaults to if you pass through null.